src/libraries/System.Security.Cryptography.Cose/tests/CoseSign1MessageTests.Verify.cs

```diff
@@ -89,6 +88,34 @@ static string ReplaceFirst(string text, string search, string replace)
             }
         }
 
+        [Fact]
+        public void VerifyReturnsTrueAfterAttempWithWrongContent()
```

Comment:
Nit: Missing the last `t` in Attempt.

```suggestion
        public void VerifyReturnsTrueAfterAttemptWithWrongContent()
```